uber/jinja.py

```diff
@@ -0,0 +1,61 @@
+from uber.common import *
+
+
+class JinjaEnv:
+    _env = None
+    _exportable_functions = {}
+    _filter_functions = {}
+    _template_dirs = []
+
+    @staticmethod
+    def append_template_dir(dirname):
+        """
+        Add another template directory we should search when looking up templates.
+        """
+        JinjaEnv._template_dirs.insert(0, dirname)
+
+    @staticmethod
+    def env():
```

Comment:
oh sorry yea that's what I meant, some of that stuff is good to put in on_startup precisely because we only want it to run when the actual web server is coming up and not for random sep commands/etc.